scripts/zkai_range_disciplined_activation_receipt.py

```
@@ -0,0 +1,464 @@
+            "not a new proof benchmark",
+            "not a transformer proof",
+            "not a JSTprove security finding",
+            "not evidence that ReLU is solved at large model scale",
+            "not a Stwo AIR result",
+        ],
+    }
+    validate_payload(payload)
+    return payload
+
+
+def validate_payload(payload: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)